app/controllers/entries_controller.rb

```diff
@@ -0,0 +1,72 @@
+class EntriesController < ApplicationController
+  before_action :authenticate_user!, only: %i[ new create edit update destroy ]
+  before_action :set_entry, only: %i[ edit update destroy ]
+
+  def index
+    @entries = Entry.all
+  end
+
+  def show
+    @entry = Entry.find(params[:id])
+  rescue ActiveRecord::RecordNotFound
+    redirect_to entries_path
+  end
+
+  def new
+    @entry = Entry.new
+  end
+
+  def edit
+  end
+
+  def create
+    @entry = Entry.new(entry_params)
+    @entry.team = current_user.team
```

Comment:
Might be better to do this?

```suggestion
    @entry = current_user.team.build_entry(entry_params)
```

Reply:
Ah yes of course lol!